mappings/net/minecraft/structure/StrongholdGenerator.mapping

```diff
@@ -256,10 +273,11 @@ CLASS net/minecraft/unmapped/C_zxumomxn net/minecraft/structure/StrongholdGenera
 			ARG 5 orientation
 			ARG 6 chainlength
 	CLASS C_soqenazm Library
-		FIELD f_auyhzmhq SIZE_Y I
-		FIELD f_flfuaoag SIZE_X I
+		FIELD f_auyhzmhq TALL_HEIGHT I
```

Comment:
`TALL_HEIGHT` and `HEIGHT` are a little weird. I would clarify what this field means if you can figure it out. From the name it seems to mean that it is the height of the top floor, so `TOP_FLOOR_HEIGHT` might be better

Reply:
Its used in `#generate` as the room height when `#tall` is true
```java
int i = TALL_HEIGHT;
if (!this.tall) {
  i = HEIGHT;
}
this.fillWithOutline(world, boundingBox, 0, 0, 0, 13, i - 1, 14, true, random, STONE_BRICK_RANDOMIZER);
```